includes/settings/class-wc-checkoutcom-cards-settings.php

```diff
@@ -290,6 +290,7 @@ public static function cards_settings() {
 					'discover'        => __( 'Discover', 'checkout-com-unified-payments-api' ),
 					'jcb'             => __( 'JCB', 'checkout-com-unified-payments-api' ),
 					'cartesbancaires' => __( 'Cartes Bancaires', 'checkout-com-unified-payments-api' ),
+					'mada'            => __( 'Mada', 'wc_checkout_com' ),
```

Comment:
Hi @ayoubAnbara 
Thanks for contribution. 
I have small feedback in this PR, Can you please update domain from `wc_checkout_com` to `checkout-com-unified-payments-api`

Once above change is added I will merge this PR to master.
Thanks

Reply:
Hi @SanketChodavadiya 
Done